CoreScriptsRoot/Modules/Server/ClientChat/ChatScript.lua

```diff
@@ -125,7 +125,7 @@ function checkBothChatTypesDisabled()
 	return false
 end
 
-if not GuiService:IsTenFootInterface() then
+if (not GuiService:IsTenFootInterface()) or (not game:GetService('UserInputService').VREnabled) then
```

Comment:
This should be and, not or. Because we only want to do this in the case where it is not Vr and not ten foot interface.